packages/bun-types/bun.d.ts

```diff
@@ -1499,6 +1499,10 @@ declare module "bun" {
 
   type ModuleFormat = "esm"; // later: "cjs", "iife"
 
+  type CompileTargetOperatingSystem = "windows" | "macos" | "linux";
```

Comment:
To make this easier for developers, what if we align this with `NodeJS.Platform`? We can also be more flexible and support aliases.